Cargo.toml

```diff
@@ -14,7 +14,8 @@ authors = ["nabijaczleweli <nabijaczleweli@gmail.com>",
            "Cat Plus Plus <piotrlegnica@piotrl.pl>",
            "Liigo <liigo@qq.com>",
            "azyobuzin <azyobuzin@users.sourceforge.jp>",
-           "Peter Atashian <retep998@gmail.com>"]
+           "Peter Atashian <retep998@gmail.com>",
+           "Daniel Abramov <dabramov@snapview.de>",]
```

Comment:
How many times have I to reiterate: don't edit my internals, especially if they're unrelated.